content/tutorial-escrow.md

```diff
@@ -19,6 +19,409 @@ When testing in [stand-alone mode](concept-stand-alone-mode.html), you can force
+
+Request:
+
+<!-- MULTICODE_BLOCK_START -->
+
+*Websocket*
+
+```json
+{% include 'code_samples/escrow/websocket/tx-request-escrowcreate-condition.json' %}
+```
+
+<!-- MULTICODE_BLOCK_END -->
+
+Response:
+
+<!-- MULTICODE_BLOCK_START -->
+
+*Websocket*
+
+```json
+{% include 'code_samples/escrow/websocket/tx-response-escrowcreate-condition.json' %}
+```
+
+<!-- MULTICODE_BLOCK_END -->
+
+### 6. Submit EscrowFinish transaction
+
+[Sign and submit](reference-transaction-format.html#signing-and-submitting-transactions) an [EscrowCreate transaction][] to execute the release of the funds after the `FinishAfter` time has passed. Set the `Owner` field of the transaction to the `Account` address from the EscrowCreate transaction, and the `OfferSequence` to the `Sequence` number from the EscrowCreate transaction. Set the `Condition` and `Fulfillment` fields to the condition and fulfillment values, in hexadecimal, that you generated in step 1. Set the `Fee` ([transaction cost](concept-transaction-cost.html)) value based on the size of the fulfillment in bytes: a conditional EscrowFinish requires at least 330 drops of XRP plus 10 drops per 16 bytes in the size of the fulfillment.
+
+**Note:** If you included a `FinishAfter` field in the EscrowCreate transaction, you cannot execute it before that time has passed, even if you provide the correct fulfillment for the Escrow's condition. The EscrowFinish transaction fails with the [result code](reference-transaction-format.html#transaction-results) `tecNO_PERMISSION` if the previously-closed ledger's close time is before the `FinishAfter` time.
+
+{% include 'snippets/secret-key-warning.md' %}
+
+<!-- MULTICODE_BLOCK_START -->
+
+_Websocket_
+
+```json
+{% include 'code_samples/escrow/websocket/submit-request-escrowfinish-condition.json' %}
+```
+
+<!-- MULTICODE_BLOCK_END -->
+
+Response:
+
+<!-- MULTICODE_BLOCK_START -->
+
+_Websocket_
+
+```json
+{% include 'code_samples/escrow/websocket/submit-response-escrowfinish-condition.json' %}
+```
+
+<!-- MULTICODE_BLOCK_END -->
+
+Take note of the transaction's identifying `hash` value so you can check its final status when it is included in a validated ledger version.
+
+### 7. Wait for validation
+
+{% include 'snippets/wait-for-validation.md' %}
+
+### 8. Confirm final result
+
+Use the [`tx` command](reference-rippled.html#tx) with the EscrowFinish transaction's identifying hash to check its final status. In particular, look in the transaction metadata for a `ModifiedNode` of type `AccountRoot` for the destination of the escrowed payment. The `FinalFields` of the object should reflect the increase in XRP in the `Balance` field.
+
+Request:
+
+```json
+{% include 'code_samples/escrow/websocket/tx-request-escrowfinish-condition.json' %}
+```
+
+Response:
+
+```json
+{% include 'code_samples/escrow/websocket/tx-response-escrowfinish-condition.json' %}
+```
+
+
```

Comment:
I didn't scope the tutorial to include `EscrowCancel` because I didn't think that it made sense to plan a task that intends to cancel the escrow. That said, I think it would be sensible to add a mini-tutorial for how to cancel an escrow, picking up partway through one of the existing tutorials. I'm adding a task to the docs backlog for that, so it doesn't have to block publishing this doc.